src/neural/loader.cc

```diff
@@ -48,14 +48,78 @@ namespace lczero {
 namespace {
 const std::uint32_t kWeightMagic = 0x1c0;
 
+uint32_t ReadLittleEndian(const uint8_t *addr) {
```

Comment:
Do we really need this function? There are already places where we rely on system being little endian..
If we really need it, then consider moving it to `utils/bititer.h`.